packages/code-du-travail-frontend/cypress/support/before.ts

```diff
@@ -0,0 +1,27 @@
+// @ts-ignore
+import fs from "fs";
+
+export const downloadAllUrlsToValidate = async () => {
+  const urls: string[] = ["/convention-collective", "/contribution"];
+
+  const response = await fetch(
+    "https://code-du-travail-numerique-preprod.dev.fabrique.social.gouv.fr/api/plan-du-site"
```

Comment:
```suggestion
    "/api/plan-du-site"
```
A changer avant de merger